### DISCUSSION TITLE
deleted columns are not recognized when fetching tables

### DISCUSSION THREAD

**Discussion by kmdelucaNG** [Budibase/budibase#6586] [Help] [answered]
I have several postgres tables that are working great.   However, when I remove a column from a table (externally using pgAdmin for example), Budibase still thinks the column is there.  Even after re-fetching the table.   New columns are showing up, but deleted columns are not "going away".
As a debugging step, if I create an entirely new data source, the correct columns are recognized.

**Comment by kmdelucaNG** [comment]
Also tried deleting the (old) columns from Budibase.  I'm allowed to delete them, but they still show up and cause problems, 
<img width="164" alt="image" src="https://user-images.githubusercontent.com/86023779/177782319-a484f796-d9bc-4f22-89a6-048f145a10ba.png">

**Comment by melohagan** [comment, marked as answer]
Hi @kmdelucaNG 

Thank you for bringing this to attention. I have raised a bug here: https://github.com/Budibase/budibase/issues/6737
